pkg/server/home_workspaces.go

```diff
@@ -270,9 +270,11 @@ func (h *homeWorkspaceHandler) ServeHTTP(rw http.ResponseWriter, req *http.Reque
 
 	ctx := req.Context()
 	logger := klog.FromContext(ctx)
-	lcluster, err := request.ValidClusterFrom(ctx)
-	if err != nil {
-		responsewriters.InternalError(rw, req, err)
+	lcluster := request.ClusterFrom(req.Context())
```

Comment:
I'll follow up with a PR that removes usage of `ValidClusterFrom` from the authorizers